python/cudf/cudf/core/index.py

```diff
@@ -437,6 +446,18 @@ def from_pandas(cls, index):
         ind.name = index.name
         return ind
 
+    @classmethod
+    def _from_table(cls, table):
+        if len(table._data) == 0:
+            raise ValueError("Cannot construct Index from any empty Table")
+        if len(table._data) == 1:
```

Comment:
Is `len(table._data)` the number of columns? Could we be more explicit about this because I assumed this was the number of rows.

Reply:
Changed to `_num_columns` (also added `_num_rows`)